docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -165,4 +165,27 @@ writes it on the data node:
     data_retrieved = data_node.read()
     ```
 
+# Get parent scenarios, pipelines and tasks
```

Comment:
Get -> Retrieving the (?)

That would be the 'User Manual' way of setting section titles.